src/activities/morse_code/MorseCode.qml

```diff
@@ -370,12 +376,81 @@ ActivityBase {
 
         }
 
+        GCText {
+            id: morse_map
+            wrapMode: TextEdit.WordWrap
+            fontSize: mediumSize
+            horizontalAlignment: Text.AlignHCenter
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+            width: parent.width * 0.8
+            height: parent.height * 0.8
+            visible: false
+            text: qsTr("\nA = .-\tB = -...\tC = -.-.\tD = -..\tE = .")
+                  + "\n" + qsTr("F = ..-.\tG = --.\tH = ....\tI = ..\t\tJ = .---")
```

Comment:
There are 2 \t just before J, is it normal?
It may be better to use a Flow with for each element you'll have the letter and its morse value. This way, they won't be elided in the display (like in http://pasteboard.co/mPLrZbZOO.png where the J is in one line and its morse code in the next one).